README.md

```diff
@@ -26,92 +26,92 @@ $ brew install --HEAD authzed/tap/zed
 
 ## Example Usage
 
-### Managing credentials
+### Logging into a permission system
 
-Configuring credentials is similar to [kubeconfig] in [kubectl].
-
-[kubeconfig]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/
-[kubectl]: https://kubernetes.io/docs/reference/kubectl/overview/
-
-API Tokens are stored in the system keychain.
+In order to interact with a permission system, zed first needs an API token.
```

Comment:
I've been using "Permissions System" (capitals) as a means to indicate it is a Defined Thing